hwtracer/src/llvm_blockmap.rs

```diff
@@ -0,0 +1,216 @@
+//! Parser for ykllvm's extended `.llvm_bb_addr_map`.
+
+use byteorder::{NativeEndian, ReadBytesExt};
+use intervaltree::IntervalTree;
+use libc::{dlsym, RTLD_DEFAULT};
+use std::{
+    convert::TryFrom,
+    ffi::CString,
+    io::{prelude::*, Cursor, SeekFrom},
+    slice,
+    sync::LazyLock,
+};
+
+pub static LLVM_BLOCK_MAP: LazyLock<BlockMap> = LazyLock::new(BlockMap::new);
+
+#[derive(Debug)]
+pub enum SuccessorKind {
+    // FIXME notes on divergence.
```

Comment:
Did you mean to leave this FIXME here?

Reply:
Oops. I was supposed to comment up this struct!

Reply:
Fixed in e137494e3af8f39d2dcb03f2fcfb77e31af7fb8d